runtime/client/src/clientde_impl.cpp

```diff
@@ -1531,8 +1531,11 @@ LTRESULT CLTClient::SetLightGroupColor(uint32 nID, const LTVector &vColor)
 	// Make sure our status is acceptable
 	if (!world_bsp_client)
 		return LT_NOTINWORLD;
-
-	if (!r_GetRenderStruct() || !r_GetRenderStruct()->SetLightGroupColor(nID,vColor))
+#ifndef __LINUX
+	if (!r_GetRenderStruct() || !r_GetRenderStruct()->SetLightGroupColor)
```

Comment:
this has to be added on windows as this is a function pointer
and you'll need to call the function if it is not a nullptr

Reply:
my bad, this is not a function pointer, but a shim function that calls the loaded render object, on linux
```
/home/travis/build/Katana-Steel/lithtech/runtime/client/src/clientde_impl.cpp: In member function ‘virtual LTRESULT CLTClient::SetLightGroupColor(uint32, const LTVector&)’:
/home/travis/build/Katana-Steel/lithtech/runtime/client/src/clientde_impl.cpp:1535:52: error: cannot convert ‘RenderStruct::SetLightGroupColor’ from type ‘bool (RenderStruct::)(uint32, const LTVector&) {aka bool (RenderStruct::)(unsigned int, const TVector3<float>&)}’ to type ‘bool’
  if (!r_GetRenderStruct() || !r_GetRenderStruct()->SetLightGroupColor)
```

Reply:
This is why I wanted to use the g_RenderStructZeroed idea. Unless there's an easy way to turn SetLightGroupColor into a function pointer.

Reply:
Made another attempt. Hopefully this should work.

Reply:
if you change it to a function pointer... you will also need to set/assign it in 

and it still doesn't exempt  you from calling SetLightGroupColor(..) inside the if block
you're changing the behavior of the function.

Reply:
SetLightGroupColor is currently only a D3D function (see d3d_SetLightGroupColor). Can this be done later? I'd just like to make other fixes to the game code before OpenGL/Vulkan replacement code is made.